Packs/Bitbucket/Integrations/Bitbucket/Bitbucket_description.md

```diff
@@ -1,14 +1,18 @@
 ## Bitbucket Integration Help
 
 * To create an app password to Bitbucket
-  1. Go to your account > **Settings** > **App Passwords**.
-  2. Click **Create app password**.
-  3. Give the app password a name related to the application that will use the password.
-  4. Select the specific access and permissions you want this application password to have.
-  5. Copy the generated password and either record or paste it into the application you want to give access. The password is only displayed this one time.
-  6. Click [here](https://developer.atlassian.com/cloud/bitbucket/rest/intro/#authentication) for more information.
+  1. Select your avatar (The icon that represent your profile and settings) from the navigation bar at the top of the screen.
+  2. Under **Settings**, select **Personal settings**.
+  3. On the sidebar, select **App passwords**.
+  4. Select **Create app password**.
+  5. Give the app password a name, usually related to the application that will use the password.
+  6. Select the permissions the App password needs. For detailed descriptions of each permission, see: [App password permissions](https://support.atlassian.com/bitbucket-cloud/docs/app-password-permissions/).
+  7. Select the **Create** button. The page will display the **New app password** dialog.
+  8. Copy the generated password and either record or paste it into the application you want to give access. ***The password is only displayed once and can't be retrieved later***.
+  9. Click [here](https://support.atlassian.com/bitbucket-cloud/docs/create-an-app-password/) for more information.
   
 ### Known Limitations
+* In order to perform the ***test-module*** command (the "Test" button) add to the **app password**, **Read** permissions to **Projects**. 
```

Comment:
Maybe just mention that the read permission is required for the projects, no need to specify test module and stuff